pkg/server/backend_manager.go

```diff
@@ -27,6 +28,29 @@ import (
 	"sigs.k8s.io/apiserver-network-proxy/proto/agent"
 )
 
+type ProxyStrategy string
+
+const (
+	ProxyStrategyDestHost ProxyStrategy = "destHost"
+	ProxyStrategyDefault  ProxyStrategy = "default"
+)
+
+// GenProxyStrategiesFromStr generates the list of proxy strategies from the
+// comma-seperated string, i.e., destHost,defualt.
```

Comment:
default

Reply:
The default strategy will always be added. And we won't let the user specify default anymore.